### DISCUSSION TITLE
Why do I get on startup I get urllib.error.URLError?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1143] [General]
When I start up VisiData with a CSV file I get an `urllib.error.URLError: ...`.

Why's that? Does `vd` want to download something from the Internet? And why?

Also, the error message disappears when I start browsing the file. Is there a way --  probably a sheet :-) -- to see the error message again?

Many thanks for any help!

**Comment by saulpw** [comment]
Yes, it downloads the motd and the list of plugins.  You can see the errors sheet with `Ctrl+E`, or the previous statuses sheet with `Ctrl+P`.

**Comment by halloleo** [comment]
Thanks, Saul. Through your (very prompt) comment I found #850. So that's I will use to disable this phoning home..

BTW, where do I see this motd? On another system, where I _do_ have Internet access, I've never been greetd with a motd...

**Reply by saulpw** [reply]
It's the little random message on the status line that appears on startup.  You can disable it if you want, of course, but if you're really concerned about privacy, you may want to avoid posting on Github, as Microsoft is certainly tracking you more than I am.  I just want to know how many people are using the software that I've put so much effort into.

**Reply by anjakefala** [reply]
See here for why the careful decision to add this was made: https://github.com/saulpw/visidata/discussions/940#discussioncomment-572821

**Reply by halloleo** [reply]
@saulpw @anjakefala Thanks for the info and the link to the detailed(!) explanation. I have no particular privacy concerns here, but in one of my environments outgoing traffic is blocked and logged, so it's better for me to disable the url fetch calls there (which is easy to do after reading #850).

I totally understand that you need usage data! Keep up the _great_ work!
